deployment/ansible/group_vars/all

```diff
@@ -30,6 +30,7 @@ postgis_package_version: "2.3.1+dfsg-1.pgdg14.04+1"
 # azavea.docker
 # 1.11.* to match AWS ECS
 docker_version: 1.11.*
+docker_options: "--storage-opt dm.basesize=20G"
```

Comment:
Yeah, I think we need to do something equivalent to:

```
$ docker run -v /mnt/${JOB_ID}:/var/lib/postgresql/data ...
```

Where `/mnt` is some larger amount of storage (ephemeral or EBS). Scoping by job ID, or something else, helps support having multiple jobs like these running alongside each other.